Assets/Scripts/Models/Prototypes/PrototypeManager.cs

```diff
@@ -22,9 +22,11 @@ public PrototypeManager()
         Inventory = new PrototypeMap<InventoryCommon>("Inventories", "Inventory");
         TileType = new PrototypeMap<TileType>("Tiles", "Tile");
         Furniture = new PrototypeMap<Furniture>("Furnitures", "Furniture");
-        FurnitureJob = new PrototypeMap<Job>();
+        FurnitureBuildJob = new PrototypeMap<Job>();
+        FurnitureDeconstructJob = new PrototypeMap<Job>();
         Utility = new PrototypeMap<Utility>("Utilities", "Utility");
-        UtilityJob = new PrototypeMap<Job>();
+        UtilityConstructJob = new PrototypeMap<Job>();
```

Comment:
This should be UtilityBuildJob or FurnitureBuildJob should be FurnitureConstructJob. Either way is fine as long as both use the same build name.